capten/agent/cmd/agent/main.go

```diff
@@ -26,7 +28,19 @@ func main() {
 		log.Fatalf("Fetching application configuration failed, %v", err)
 	}
 
-	s, err := agent.NewAgent(log)
+	//Todo: read from env var
+	store := cassandra.NewCassandraStore(log, nil)
+	err = store.Connect([]string{"localhost:9042"}, "cassandra", "cassandra")
+	if err != nil {
```

Comment:
Used Option pattern to create cassandra and temporal inside Agent so that we can create require config in isolation for testing, a better way to inject dependencies

Reply:
also marked some of the ENV vars as `required:false`